docs/Build/grestsvcs.json

```diff
@@ -569,7 +577,7 @@
       }
     },
     "/epoch_info_cache": {
-      "get": {
+      "post": {
```

Comment:
GET works, prolly preferable too - something I'd try and update for others - where RETURN is table